### PR TITLE
Add support for custom vCPU configuration in RHV platform

### PR DESCRIPTION
**Pull request by sidhant-agrawal**
Closes: #4442 

Signed-off-by: Sidhant Agrawal <sagrawal@redhat.com>

### CODE REVIEW

File: ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2 [hunk 1]
```diff
@@ -7,8 +7,8 @@ compute:
   platform:
     ovirt:
       cpu:
-        cores: {{ worker_num_cpus | default(4) }}
-        sockets: 1
+        cores: {{ worker_num_cores | default(4) }}
+        sockets: {{ worker_num_sockets | default(1) }}
```
Comment (vasukulkarni) [inline ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2:11]: Doesn't this change the default for other implementation and set it to 1? I think better to set this in RHV only block.

File: ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2 [hunk 2]
```diff
@@ -21,8 +21,8 @@ controlPlane:
   platform:
     ovirt:
       cpu:
-        cores: {{ master_num_cpus | default(4) }}
-        sockets: 1
+        cores: {{ master_num_cores | default(4) }}
+        sockets: {{ master_num_sockets | default(1) }}
```
Comment (vasukulkarni) [inline ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2:25]: same as above

File: ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2 (hunk 1 above)
Comment (sidhant-agrawal) [reply ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2:11]: This template file `install-config-rhv-ipi.yaml.j2` and other conf files in this PR are specific to RHV platform.
So it shouldn't affect other platform implementations.

**Comment by petr-balogh** [comment]
@sidhant-agrawal is this ready for review merge? I see it's verified already.

**Comment by sidhant-agrawal** [comment]
@petr-balogh yes, it's already verified.

**Review by petr-balogh — APPROVED** [review]

**Review by clacroix12 — APPROVED** [review]

**Pull request merged**